libs/blocks/accordion/accordion.js

```diff
@@ -35,17 +78,27 @@ function createItem(accordion, id, heading, num) {
   }, heading.textContent);
   button.append(icon);
 
-  const panel = heading.nextElementSibling.firstElementChild;
-  const para = panel.querySelector('p');
-  const text = para ? para.textContent : panel.textContent;
+  const panel = heading.nextElementSibling?.firstElementChild;
+
+  const para = panel?.querySelector('p');
+  const text = para ? para.textContent : panel?.textContent;
 
   const dtAttrs = hTag ? {} : { role: 'heading', 'aria-level': 3 };
   const dtHtml = hTag ? createTag(hTag.tagName, { class: 'accordion-heading' }, button) : button;
   const dt = createTag('dt', dtAttrs, dtHtml);
   const dd = createTag('dd', { role: 'region', 'aria-labelledby': triggerId, id: panelId, hidden: true }, panel);
+  const dm = createTag('dd', { class: 'media-p' });
 
-  button.addEventListener('click', (e) => { handleClick(e.target, dd); });
+  if (edit) {
+    const ogMedia = mediaCollection[id][num - 1];
+    const mediaCopy = ogMedia.cloneNode(true);
+    dm.append(mediaCopy)
+    dd.prepend(dm);
```

Comment:
Nit: This seems odd to nest a `<dd>` directly inside of the parent `<dd>`. Not sure if this will cause accessibility or screenreader issues. Changing this element to a `<div>` similar to the other nest content makes more sense.

![image](https://github.com/adobecom/milo/assets/2539954/3f503dc9-6725-4973-a4a3-574fb152da4c)

Couldn't you just directly add the `<picture>` instead of creating a new element to put it in?

Reply:
went with the `div` it's slightly easier to display the section on on mobile/tablet this way